DeusEx/Classes/DXRHordeMode.uc

```diff
@@ -188,17 +223,20 @@ function AnyEntry()
     local DeusExMover d;
     local DXREnemies dxre;
     local Inventory item;
+    local int i;
 
     if( dxr.flags.gamemode != 2 ) return;
 
-    if( dxr.dxInfo.missionNumber>0 && dxr.localURL != "11_PARIS_CATHEDRAL" ) {
-        Level.Game.SendPlayer(dxr.player, "11_PARIS_CATHEDRAL");
+    if( dxr.dxInfo.missionNumber>0 && dxr.localURL != map_name ) {
+        Level.Game.SendPlayer(dxr.player, map_name);
         return;
     }
-    else if( dxr.localURL != "11_PARIS_CATHEDRAL" ) {
+    else if( dxr.localURL != map_name ) {
         return;
     }
 
+    dxr.Player.SetLocation(starting_location);
```

Comment:
should I do `if( starting_location != vect(0,0,0) )`? or would that be confusing for people who try to mess with the configs?